tests/codegen_testcases/slice1.sol

```diff
@@ -4,14 +4,15 @@ contract c {
 	function test1() public pure {
 		bytes x = "foo1";
 		// x is not being used, so it can be a slice
-// CHECK: alloc slice uint32 4 "foo1"
+// CHECK:
```

Comment:
@seanyoung Do you have any suggestion on how I should fix this test? I tried returning bytes, but it didn't work.

Reply:
Yes, returning bytes will make it into a slice (this needs fixing, but needs an analysis pass across functions).

You can return a subscript:
```
        function test1() public returns (byte) {
                bytes x = "foo1";

                return x[0];
        }
```